kapitan/cli.py

```diff
@@ -275,7 +275,10 @@ def main():
             # update recipients for all secrets in secrets_path
             # use --secrets-path to set scanning path
             inv = inventory_reclass(args.inventory_path)
-            targets = set(inv['nodes'].keys())
+            if args.target_name:
+                targets = set(args.target_name.split(','))
```

Comment:
I thought `target_name` has a different purpose according to the [description](https://github.com/deepmind/kapitan/blob/f08caef6a3694642baec43298c634b82670a419d/kapitan/cli.py#L132). Maybe add `--targets` flag for that. Similar to [this](https://github.com/deepmind/kapitan/blob/f08caef6a3694642baec43298c634b82670a419d/kapitan/cli.py#L74).